src/Veldrid/OpenGL/OpenGLExtensions.cs

```diff
@@ -1,16 +1,18 @@
-﻿using System;
+﻿using System.Collections;
 using System.Collections.Generic;
 
 namespace Veldrid.OpenGL
 {
-    internal class OpenGLExtensions
+    public class OpenGLExtensions : IReadOnlyCollection<string>
```

Comment:
Again, I'm not sure about exposing this whole type publically. The raw list of extensions should be available, but all of the other helper members on this type aren't a good idea to expose IMO.